src/components/EventList.tsx

```diff
@@ -59,9 +62,20 @@ const EventList = ({ events, org } : EventListProps) : JSX.Element => {
                             />
                         </View>
                         <View data-test="location-title">{ e.location.title }</View>
-                        <Button data-test="sign-up-button" marginTop="size-50" variant="cta">
+                        <Button
+                            data-test="sign-up-button"
+                            marginTop="size-50"
+                            onPress={ () => putEventResponse(e.id, org?.id) }
```

Comment:
Great! This is exactly what an event handler should look like.

But why can `org` be `undefined` here? That should never be the case.

Reply:
Perfect! In `EventListProps` we've previously defined that `org` can be `undefined`. I have a vague memory of us leaving it like this since `events` and hence `EventList` will never run  if we in `events` don't get `orgState?.status === 'success'` in `getServerSideProps`.

Reply:
If `org` will never be undefined, it should not be typed as potentially being `undefined`.

If we know that `org` will never be `undefined` we should use non-null assertion or conditionals to indicate that to typescript. Otherwise we're just sweeping the problem in front of us.

This line is essentially saying that if `org` is undefined, pass `undefined` to `putEventResponse()`. What happens if we do that? The function might not work as expected.

Reply:
Yes, this sounds reasonable. Does the same go for `events` (non-null assertion or conditionals)?

Reply:
I don't understand the question. It looks like `e` is already guaranteed to have a value here, so no changes should be required there.